appendix.Rmd

```diff
@@ -77,4 +83,15 @@ knitr::write_bib(pkgs, file="packages.bib")
 knitr::kable(pkgs_df)
 ```
 
+These can be installed as follows:
+
+```{r, eval=FALSE}
+u = "https://git.io/vX8aa"
```

Comment:
You could always add a comment above install_github.

Alternatively (and perhaps cleaner), just add dependencies on the csgillespie/efficient package